libpromises/unix.c

```diff
@@ -202,6 +203,19 @@ bool ShellCommandReturnsZero(const char *command, ShellType shell)
         else
         {
             char **argv = ArgSplitCommand(command);
+            int devnull;
+
+            if ((devnull = safe_open("/dev/null", O_WRONLY)) == -1)
```

Comment:
Btw, disregard this, apparently there is a separate Windows implementation of this function.